product_configurator_wizard/wizard/product_configurator.py

```diff
@@ -219,12 +274,13 @@ def onchange(self, values, field_name, field_onchange):
         comodel_name='product.product',
         readonly=True,
         string='Product Variant',
-        help='Set only when re-configuring a existing variant'
+        help='Set only when re-configuring an existing variant'
     )
     product_img = fields.Binary(
         compute='_compute_cfg_image',
         readonly=True
     )
+    stored_support_vals = fields.Char()
```

Comment:
If we need to compute the values of the stored fields on each step and we need them only there in that view, do we need to store them like the other values?